source/indexes.txt

```diff
@@ -97,8 +95,6 @@ Atlas Vector Search indexes enable you to index vector data *and* other data
 types, which facilitates semantic searches on the indexed fields. For more
 information, see :ref:`compass-create-vector-search`. 
```

Comment:
this ref and the ref above will link to the corresponding Atlas Search & Vector Search Index creation pages.

Reply:
Note that this link will be swapped out in https://github.com/10gen/cloud-docs/pull/6554 since the change will only pertain to Data Explorer